Tests/AsyncKitTests/ConnectionPoolTests.swift

```diff
@@ -13,92 +13,98 @@ final class ConnectionPoolTests: XCTestCase {
         let pool = EventLoopConnectionPool(
             source: foo,
             maxConnections: 2,
-            on: EmbeddedEventLoop()
+            on: self.group.any()
```

Comment:
vapor changes.
`self.group.any()` every day all day instead of `EmbeddedEventLoop()`, even in tests.